File: src/main/java/me/delivery/domain/address/user/repository/UserAddressRepository.java
package me.delivery.domain.address.user.repository;

import me.delivery.domain.address.user.model.entity.UserAddress;
import org.springframework.data.jpa.repository.JpaRepository;

import java.util.List;

public interface UserAddressRepository extends JpaRepository<UserAddress, Long> {
    List<UserAddress> findByUserIdAndDeletedAtIsNull(long userId);

}

File: src/main/kotlin/me/delivery/domain/address/user/repository/UserAddressRepository.kt
package me.delivery.domain.address.user.repository

import me.delivery.domain.address.user.model.entity.UserAddress
import org.springframework.data.jpa.repository.JpaRepository

interface UserAddressRepository : JpaRepository<UserAddress, Long> {
    fun findByUserIdAndDeletedAtIsNull(userId: Long): List<UserAddress>
}